config/locales/en.yml

```diff
@@ -150,6 +150,8 @@ en:
     errors:
       messages:
         blank: "is required"
+        invalid_format: "isn't valid"
+        invalid_domain: "you have entered can't be used to access People Finder"
```

Comment:
I see that we have them mixed throughout the file. Is this for sure the correct one? Cause I'll fix all of them if that's the case.